examples/hardhat/test/erc20.test.js

```diff
@@ -1,7 +1,12 @@
 /* External Imports */
-const { ethers } = require('hardhat')
-const { expect } = require('chai')
+const { ethers, network } = require('hardhat')
+const chai = require('chai')
+const { solidity } = require('ethereum-waffle')
+const chaiAsPromised = require('chai-as-promised')
```

Comment:
Actually `chai-as-promised` should not be necessary here. We can probably remove this dependency.

Reply:
@platocrat can you quickly check whether removing this dependency causes any issues?